docs/src/components/landing/Hero.js

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import IconImage from 'docs/src/components/icon/IconImage';
+import GradientText from './GradientText';
+import SignUp from './SignUp';
+import ToolpadHeroContainer from '../../layouts/ToolpadHeroContainer';
+import Demo from './Demo';
```

Comment:
Since we are not using absolute import paths, I think that we should have file name as long as possible, otherwise, it will be a pain to find who is importing Demo (there could be others in the codebase)

```suggestion
import HeroDemo from './HeroDemo';
```

Reply:
One of the reasons why I prefer to avoid custom aliasing, it makes it easy for all other developer tools (like vscode) to statically analyze the code and build a dependency tree of the code files:

https://user-images.githubusercontent.com/2109932/186153317-23a1de5a-43fb-4e1c-8b21-d8e101e49e6e.mov

no extra configuration needed, it just works out of the box

Reply:
Interesting, I'm used to searching the whole filename, e.g. `import Link from 'docs/src/modules/components/Link';` in the codebase. First time I see this reference feature. I didn't know this exsited.

Reply:
I will change the names regardless, although this is a nifty feature that I use (sometimes) as well